src/main/java/ContinuousIntegrationServer.java

```diff
@@ -18,8 +18,8 @@
 */
 public class ContinuousIntegrationServer extends AbstractHandler
 {
-    private static final String gitHubRepoHTTPS = "https://github.com/myhelmisaari/DD2480-Big-Brain-CI.git";
-    private static final String assessmentRepo = "assessmentDir/";
+    public static final String gitHubRepoHTTPS = "https://github.com/myhelmisaari/DD2480-Big-Brain-CI.git";
+    public static final String assessmentRepo = "assessmentDir/";
```

Comment:
I think should be `protected`, not public.

Reply:
It should be private but for the tests need to have access to those variables